### DISCUSSION TITLE
unknown command "<container name>" for "watchtower" (new behavior in 1.5.0?)

### DISCUSSION THREAD

**Discussion by natonic77** [containrrr/watchtower#1456] [General] [answered]
Posting here since there isn't a discussion thread for 1.5.0 yet.

I tried upgrading from the watchtower 1.4.0 container to 1.5.0 this morning and am finding a fairly simple invocation that filters on a specific container name does not appear to be supported in 1.5.0.

My command is fairly simple: `docker run -d --name watchtower -v /var/run/docker.sock:/var/run/docker.sock -v /etc/localtime:/etc/localtime containrrr/watchtower:latest --schedule "0 0 4 * * *" container_to_watch`.

Regardless of which container I want to monitor, the result is always the same: `Error: unknown command "container_to_watch" for "watchtower"`.

I've also tried this without the `--schedule "0 0 4 * * *"` arguments and the behavior is the same.  I can stick with 1.4.0 for now, but I'm wondering if there is just a change in 1.5.0 syntax that I overlooked.

**Comment by piksel** [comment, marked as answer]
Thanks for the report! This was accidental and I am pushing a fix for this ASAP!

Created an issue in #1457
